content/docs/reference-events.md

```diff
@@ -151,63 +151,60 @@ boolean shiftKey
 number which
 ```
 
-The `key` property can take any of the values documented in the [DOM Level 3 Events spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
+`key` प्रॉपर्टीज, [DOM स्तर 3 इवेंट्स विनिर्देश](https://www.w3.org/TR/uievents-key/#named-key-attribute-values) में प्रलेखित किसी भी वैल्यू को ले सकती है।
```

Comment:
प्रॉपर्टीज should be property, and DOM Level 3 Events spec could be left as is in english